### ISSUE TITLE
Loinc 72488-0:  "Beta 2 glycoprotein 1 IgG and IgM panel - Serum"

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4630]
Suggest creating a new term for Loinc: 72488-0 [Beta 2 glycoprotein 1 IgG and IgM panel - Serum] 
New term label:Beta 2 glycoprotein 1 antibody positivity
New term comment (if any):Parent term Autoimmune antibody positivity HP:0030057
Your biocurator ID for loinc2hpo (if desired): OHSU:jpgourdine

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0032376
I do not think we need to have separate terms for IgM or IgG since these are autoantibodies.
Please reopen if there is a need for separate terms.
@kingmanzhang
